planner/core/testdata/integration_serial_suite_out.json

```diff
@@ -1437,6 +1437,94 @@
       }
     ]
   },
+  {
+    "Name": "TestMppJoinDecimal",
+    "Cases": [
+      {
+        "SQL": "desc format = 'brief' select t1.c1, t1.c2, t2.c1, t2.c2, t2.c3 from t t1 join t t2 on t1.c1 + 1 = t2.c2 - 10 and t1.c1 * 3 = t2.c3 / 2",
```

Comment:
1. would the expressions include more than one decimal type data, for example : t1.c1*t1.c2=t2.c1/t2.c2;
2. add some test about (anti-)semi join? may be from in() or exist().
3. what about if these expressions occur in left/right/other join conditions?

Reply:
The first two has been added. For the third one, decimal with equal operation must be an equal condition. For non-equal conditions, it's no bessiness of mpp, you will not see any difference using mpp mode.